base-notebook/start.sh

```diff
@@ -95,14 +95,11 @@ if [ $(id -u) == 0 ] ; then
         echo "$NB_USER ALL=(ALL) NOPASSWD:ALL" > /etc/sudoers.d/notebook
     fi
 
-    # Add $CONDA_DIR/bin to sudo secure_path
-    sed -r "s#Defaults\s+secure_path=\"([^\"]+)\"#Defaults secure_path=\"\1:$CONDA_DIR/bin\"#" /etc/sudoers | grep secure_path > /etc/sudoers.d/path
```

Comment:
If the user wants to allow the unprivileged `jovyan` user to become root by specifying `GRANT_SUDO=yes`, will they still be able to use `sudo conda install` in the container after this change?

Reply:
Also, if I'm missing how the switch to `gosu` removes the need for the `GRANT_SUDO=yes` flag allowing the `jovyan` user in the container to `sudo`, please let me know. As it stands, it looks like the gosu change only impacts how we initially switch to the `jovyan` user when the container starts as root, but the secure path issue is really in place to allow `sudo` back to root later one.

Reply:
Thank you for your comment.
Now it worked with `sudo conda install` by adding `conda` symbolic link to `/usr/local/bin/`
```bash
ln -t /usr/local/bin/ -s \
  $CONDA_DIR/bin/conda \
  $CONDA_DIR/bin/pip \
  $CONDA_DIR/bin/python
```
In my opinion, this way is more secure than put conda bin to `secure_path`.